mail-filter/sqlgrey/sqlgrey-1.8.0.ebuild

```diff
@@ -0,0 +1,252 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit systemd
+
+DESCRIPTION="A postfix policy service implementing a grey-listing policy"
+HOMEPAGE="http://sqlgrey.sourceforge.net/"
+SRC_URI="mirror://sourceforge/sqlgrey/sqlgrey-1.8%20%28stable%29/${P}.tar.gz"
```

Comment:
Replace that with the direct `downloads.sourceforge...` URI, this mirror no longer lists mirrors.